src/Workspaces/CoreTest/SolutionTests/SolutionTests.cs

```diff
@@ -1554,6 +1554,57 @@ public void TestWithSyntaxTree()
             var root = newTree.GetRoot();
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.Workspace)]
+        [WorkItem(3705, "https://github.com/dotnet/roslyn/issues/3705")]
+        public void TestAnalyzerConfigOptions()
+        {
+            // get one to get to syntax tree factory
+            var dummyProject = CreateNotKeptAliveSolution().AddProject("dummy", "dummy", LanguageNames.CSharp);
```

Comment:
I think there's also a CreateSolution() that isn't this call -- I think this one is for some special memory assertions...

Reply:
(if you copied from the previous test, that's because that test is doing fancy memory things)